README.md

```diff
@@ -65,7 +65,7 @@ adding the following to the dependencies section of the project's pom.xml file.
 
 ```xml
      <dependency>
-       <groupId>com.github.jmchilton.blend4j</groupId>
+       <groupId>org.galaxyproject.blend4j</groupId>
```

Comment:
Since we aren't changing the namespace, at least for this release, can you leave as `com.github.jmchilton`?